tests/processor/test_processor.py

```diff
@@ -73,6 +75,29 @@ def test_params_missing_required(self):
         with self.assertRaisesRegex(Exception, 'is a required property'):
             DummyProcessorWithRequiredParameters(workspace=self.workspace)
 
+    def test_params_preset_resolve(self):
+        with pushd_popd(tempdir=True) as tempdir:
+            with mock.patch.dict(environ, {'XDG_DATA_HOME': str(tempdir)}):
+                path = Path(tempdir) / 'ocrd-resources' / 'ocrd-dummy'
+                path.mkdir(parents=True)
+                path = str(path / 'preset.json')
+                with open(path, 'w') as out:
+                    # it would be nicer to test some existing processor which does take params
```

Comment:
We could install the processors in `tests.data` (temporarily) which output the parameters they were passed. That way, we could test not only the return value but also the output.